app/styles/proposal.scss

```diff
@@ -7,87 +7,88 @@
 
 $font-color: white;
 
-h1, h2, h3, h4, h5 {
-    color: $font-color;
-}
+.proposal-list {
 
-span.glyphicon {
-    font-size: 1.0em !important;
-}
+    h1, h2, h3, h4, h5 {
+        color: $font-color;
+    }
 
-.btn-sm {
-    margin: 1px;
-}
+    span.glyphicon {
+        font-size: 1.0em !important;
+    }
 
-.prop-header {
-    background-color: #4591d3;
-}
+    .btn-sm {
+        margin: 1px;
+    }
 
-.prop-button {
-    text-align: center !important;
-    min-height: 40px;
-}
+    .prop-header {
+        background-color: #4591d3;
+    }
 
-.prop-header-text {
-    min-height: 40px;
-    display: flex;
-    align-items: center
-}
+    .prop-button {
+        text-align: center !important;
+        min-height: 40px;
+    }
 
-.prop-text {
-    min-height: 2em;
-    margin: 0.6em;
-}
+    .prop-header-text {
+        min-height: 40px;
+        display: flex;
+        align-items: center;
+    }
 
-.prop-text-oneliner {
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    margin: auto;
-    height: 1.6em;
-}
+    .prop-text {
+        min-height: 2em;
+        margin: 0.6em;
+    }
 
-.prop-row {
-    @extend .row;
-    margin: 0;
-    display: flex;
-    align-items: center;
-}
+    .prop-text-oneliner {
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        margin: auto;
+        height: 1.6em;
+    }
 
-.list-group-item {
-    border-color: #4591d3;
-}
+    .prop-row {
+        @extend .row;
+        margin: 0;
+        display: flex;
+        align-items: center;
+    }
 
-.list-group-item:first-child {
-    border-top-left-radius: 8px;
-    border-top-right-radius: 8px;
-}
+    .list-group-item {
+        border-color: #4591d3;
+    }
 
-.list-group-item:last-child {
-    border-bottom-left-radius: 8px;
-    border-bottom-right-radius: 8px;
-}
+    .list-group-item:first-child {
+        border-top-left-radius: 8px;
+        border-top-right-radius: 8px;
+    }
 
-.prop-dropdown-item {
-    padding-left: 20px;
-    padding-right: 20px;
-    padding-top: 5px;
-    padding-bottom: 5px;
-    a {
-        color: black;
+    .list-group-item:last-child {
+        border-bottom-left-radius: 8px;
+        border-bottom-right-radius: 8px;
     }
-}
 
-.glyphicon { font-size: 20px !important; }
+    .prop-dropdown-item {
+        padding-left: 20px;
+        padding-right: 20px;
+        padding-top: 5px;
+        padding-bottom: 5px;
+        a {
+            color: black;
```

Comment:
Would be nice if there was a way to have constants for header/body/dropdown eyc. background and font colors, so changing styles was easy.
Maybe future changes should be built like that whenever reasonable.